maker-core/list-of-scopes.md

```diff
@@ -1,32 +1,16 @@
-3. **RWA Collateral:** This scope covers the onboarding, management, and offboarding of Real World Assets collateral. It also covers the onboarding of new [Peg Stability Modules](https://manual.makerdao.com/module-index/module-psm). It includes parameter adjustments, prioritization of new collateral types, onboarding work for new collateral types, vault adoption, and D3M connections to Protector SubDAOs. 
-4. **Ecosystem Operations:** This scope handles the governance processes of Maker Core. It also covers governance communications and the management of community platforms. It is also responsible for SubDAO monitoring and initial dispute resolution for SubDAOs.
- 
-## Supporting Scopes
-
-The following scopes are the supporting scopes of the Endgame Constitution.
-5. **Protocol Engineering:** This scope covers all aspects of upgrading and maintaining the Maker Core protocol. It includes research, development, and deployment of protocol-level products, features, and upgrades. It also includes security-related work such as continuous audits of smart contracts, security research, and bug bounty programs.
-6. **Finance:** This scope covers the financial reporting processes and systems for the protocol. It also provides regular comprehensive updates and overviews of the state of the protocol's finances.
-7. **Physical Resilience:** This scope identifies and mitigates legal, legislative, and regulatory threats to the protocol.
-8. **Interface:** This scope focuses on the regulation and development of the core elements of the Easy Governance Frontend for the Maker community. It is responsible for a fully functional and secure decentralized frontend codebase that is available for the workforce and users to interface with the Maker Protocol.
-9. **Infrastructure:** This scope handles the security requirements of the most critical infrastructure of the Maker Protocol. This includes oracles, keepers, and other core infrastructure.
-10. **Growth:** This scope focuses on developing the Maker and Dai brands. It allocates the budget to fund various growth and marketing initiatives. It is also responsible for maintaining the official MakerDAO and SubDAO websites, frontend designs, and branding.
+1. **Stability** This scope covers the management of the Dai Stablecoin. The Dai Stablecoin must be a permissionless and useful currency available to anyone. Its stability and risk must be managed to generate as much value for MakerDAO and public good as possible.
+2. **Support:** This scope covers rules that regulate various tasks of ecosystem support, including governance process infrastructure and management, SubDAO and Ecosystem Actor support, Public Good Purpose System.
+3. **Protocol:** This scope covers technical rules and processes for the development, maintenance, and security of the Protocol.
```

Comment:
I would copy over from the Atlas. The first paragraph under each scope can be copy pasted here. If you prefer, don't include the NewChain and Gravity Bridge stuff

Reply:
Please also copy over from the Atlas for the other Scopes